platform/android/detect.py

```diff
@@ -34,8 +34,13 @@ def get_opts():
             "android-" + str(get_min_target_api()),
         ),
         BoolVariable("store_release", "Editor build for Google Play Store (for official builds only)", False),
-        BoolVariable("generate_apk", "Generate an APK/AAB after building Android library by calling Gradle", False),
+        BoolVariable(
+            "generate_android_binaries",
+            "Generate APK, AAB & AAR binaries after building Android library by calling Gradle",
+            False,
+        ),
         BoolVariable("swappy", "Use Swappy Frame Pacing library", False),
+        BoolVariable("do_not_strip_android_so", "Whether to strip the Android *.so libraries or not", False),
```

Comment:
Maybe name it `gradle_do_not_strip` which is a tiny bit shorter and makes it clear that it only works when calling gradle (i.e. via `generate_android_binaries`)?